app/src/main/java/com/rockthevote/grommet/data/db/Converters.kt

```diff
@@ -18,4 +23,50 @@ class Converters {
         return date?.time?.toLong()
     }
 
+    @TypeConverter
+    fun fromPartnerVolunteerText(volunteerText: PartnerVolunteerText): String? {
+        val adapter = PartnerVolunteerText.jsonAdapter(Moshi.Builder().build())
```

Comment:
`Moshi.Builder().build()` is pretty expensive IIRC, it might be good to instantiate it as a class level `val`

Reply:
hmm.. it's a util class though, so how would that work?